Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -45,7 +45,7 @@ - (void)setupNavigationController
     vc.filter = self.filter;
     vc.allowMultipleSelection = self.allowMultipleSelection;
     if (!self.dataSource) {
-        self.dataSource = [self defaulDataSource];
+        self.dataSource = [WPPHAssetDataSource sharedInstance];
```

Comment:
Does this need to be a singleton? Can we not just create a new one? I guess the issue is that `dataSource` is a weak reference?